tokens/0x80238d9451f86dcf12bff21af9e8deadcf915bff.yaml

```diff
@@ -0,0 +1,14 @@
+---
+addr: '0x80238d9451f86dcf12bff21af9e8deadcf915bff'
+decimals: 18
+description: >-
+  Being an intermediary between various entities involved in the ticket reselling process,
+  the platform serves as a guarantor of transaction security, tickets’ validity, timely payment and delivery.
+links:
+- Website: https://et4.io
```

Comment:
This [is failing](https://travis-ci.org/forkdelta/tokenbase/builds/437782448#L575) with an `SSLError`. This is a test problem, I'll see if I can do anything about it.